corehq/apps/smsbillables/models.py

```diff
@@ -261,7 +261,8 @@ class SmsBillable(models.Model):
     calculating the SmsLineItem in the monthly Invoice.
     """
     gateway_fee = models.ForeignKey(SmsGatewayFee, null=True, on_delete=models.PROTECT)
-    direct_gateway_fee = models.DecimalField(null=True, max_digits=10, decimal_places=4)
+    # we have observed prices with up to 5 decimal places from Twilio
+    direct_gateway_fee = models.DecimalField(null=True, max_digits=10, decimal_places=5)
```

Comment:
Should `max_digits` be increased to 11? I'm guessing it probably does not matter, but this change is reducing the largest number than can be stored by a factor of 10.

Reply:
So did some queries on production to see what how large an SmsBillable's `direct_gateway_fee` can be. The largest one is `1.35` and it is rare that they are > 1.0. Are there other environments that would be important to check or is that convincing enough that this decrease of a factor of 10 will not cause any issues?

Reply:
I'm guessing India and ICDS are similar to prod, but don't know for sure. There are certainly environments outside of our control that we cannot check, but it's probably not necessary. I brought it up mainly to make sure you had thought about it.

Here's my thinking: I doubt any environments have gateway fees >= 100K. However, are the units always $US? I'm guessing it usually is, but not sure. I suppose other currencies could have different common value ranges (still 100K seems way high). The main reason I was bringing it up was because there is a real difference in the numbers that can be represented, and it might be easier to change it to `max_digits=11, decimal_places=5`, but I'm not sure what effect that has on the migration (see other comment on migration time/rewriting whole table).